extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -1218,4 +1212,32 @@ export class TargetSelectionPage extends MigrationWizardPage {
 			targetDatabaseCollation.length > 0 &&
 			sourceDatabaseCollation.toLocaleLowerCase() === targetDatabaseCollation.toLocaleLowerCase();
 	}
+
+	private _updateNextButton() {
+		if (this.migrationStateModel.tdeMigrationConfig.shouldAdsMigrateCertificates() &&
+			this.migrationStateModel.tdeMigrationConfig.lastTdeMigrationResult().state !== TdeMigrationState.Succeeded) {
+			this.wizard.pages[this.wizard.currentPage + 1].enabled = false;
+			this.wizard.nextButton.enabled = false;
+		} else {
+			this.wizard.pages[this.wizard.currentPage + 1].enabled = true;
+			this.wizard.nextButton.enabled = true;
```

Comment:
does this work?  I have not been able to disable next button this way, typically this status is checked in the onPageLeave event which returns a validation failure message when invalid/blocked.

Reply:
Yes this works.  I needed to both disable next button and disable next page.